src/fswAlgorithms/attControl/mrpProportionalDerivative/mrpProportionalDerivative.cpp

```diff
@@ -96,3 +96,48 @@ void MrpProportionalDerivative::UpdateState(uint64_t callTime)
     eigenVector3d2CArray(Lr, controlOutMsg.torqueRequestBody);
     this->cmdTorqueOutMsg.write(&controlOutMsg, moduleID, callTime);
 }
+
+/*! Getter method for the derivative gain P.
+ @return const double
+*/
+const double MrpProportionalDerivative::getDerivativeGainP() const {
```

Comment:
:small_red_triangle_down: A `const` declaration has no effect on types like double, float, int etc. This can be removed.

Reply:
Done